lib/xccdf2inspec/xccdf2inspec.rb

```diff
@@ -24,9 +25,12 @@ def initialize(xccdf_path, cci_path, output, output_format, seperated, replace_t
     @seperated = false if seperated == 'false'
     @replace_tags = replace_tags.split(',').map(&:strip) unless replace_tags.nil?
     @controls = []
+    @mappingxls = '' if mapping_xls.nil?
+    @mappingxls = mapping_xls unless mapping_xls.nil?
+    @verbose = verbose
     replace_tags_in_xml unless replace_tags.nil?
     parse_xmls
-    parse_controls
+    parse_controls(verbose, mapping_xls)
```

Comment:
We don't need to have these as parameters since they are class instances denoted by the '@'